charts/gitops-runtime/templates/_components/internal-router/_deployment.yaml

```diff
@@ -38,20 +38,20 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
             - name: http
-              containerPort: 80
+              containerPort: 8080
               protocol: TCP
           readinessProbe:
             initialDelaySeconds: 10
             timeoutSeconds: 10
             httpGet:
-              port: 80
+              port: http
```

Comment:
I think this and the livenessProbe should also be 8080 now.
Also, why are you changing this only for the internal-router?

Reply:
@ilia-medvedev-codefresh

Reply:
Liveness probe works by port name. If you look at the definition of the port here, you see the port is 8080 but the name is http https://github.com/codefresh-io/gitops-runtime-helm/blob/b0198ee8402129041703e08c0de53eb5f891db7f/charts/gitops-runtime/templates/_components/internal-router/_deployment.yaml#L40C18-L40C18. We are changing it for internal-router because we are switching to unprivileged Nginx to be able to run on rootless (and Openshift). The rest of the components already run rootless

Reply:
Oh by bad, I didn't notice that. Thanks :)